### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by POWERCHORD77ROCKS** [TheSchoolMediaticArchive/TheSchoolMediaticArchive.github.io#6] [General]
whatssup chat its jaden and im having trouble opening and accessing games if anyone can help that would be nice

**Comment by TheEnginnerFromTF2** [comment]
you can use this list


1. [Emerald](https://theschoolmediaticarchive.github.io/CDG/Emerald)
2. [Ultrakill](https://theschoolmediaticarchive.github.io/CDG/Ultrakill)
3. [ReactorGame](https://theschoolmediaticarchive.github.io/CDG/ReactorGame)
4. [Ball](https://theschoolmediaticarchive.github.io/CDG/Ball)

**Comment by POWERCHORD77ROCKS** [comment]
https://replit.com/@jscraft/sm64
